back/spec/services/participants_service_spec.rb

```diff
@@ -120,45 +209,6 @@
       expect(service.projects_participants([project]).map(&:id)).to match_array participants.map(&:id)
     end
 
-    it 'returns correctly cached total project participant count including anonymous posts & everyone surveys' do
-      project = create(:project)
-      pp1, pp2, pp3, pp4, pp5 = create_list(:user, 5)
-
-      # Normal participation - +3
-      idea1 = create(:idea, project: project, author: pp1)
-      idea2 = create(:idea, project: project, author: pp2)
-      create(:idea, project: project, author: pp3)
-      create(:comment, post: idea1, author: pp3)
-      create(:comment, post: idea2, author: pp3)
-      create(:comment, post: idea2, author: pp2)
-      expect(service.project_participants_count(project)).to eq 3
```

Comment:
Reorganized this file a bit, there were a lot of odd things like the `project_participants_count` being tested under the `projects_participants` tests